examples/case_studies/bart_heteroscedasticity.ipynb

```diff
@@ -0,0 +1,505 @@
+{
```

Comment:
<p>The fit looks good! In fact, we see that the mean and variance increase as a function of the budget.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/519/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
Changed in https://github.com/pymc-devs/pymc-examples/pull/519/commits/6dc6186878cdfeaf04eed204ff14536349aa0502